tests/test_react_auth_defs.py

```diff
@@ -52,11 +55,45 @@
 DOMAIN = "some-domain"
 # Mock out the call which create_authtoken makes Envs.get_user_auth_info
 # to get the allowed environments and first/last name for the user.
-ENVS.get_user_auth_info = lambda email: (ALLOWED_ENVS, FIRST_NAME, LAST_NAME)
+ENVS.get_user_auth_info = lambda email, raise_exception: (ALLOWED_ENVS, FIRST_NAME, LAST_NAME)
 
 AUTH = Auth(AUTH0_CLIENT_ID, AUTH0_SECRET, ENVS)
 
 
+class MockChaliceApp:
+
+    class MockReactApi:
+        def __init__(self):
+            self.react_authorize = AUTH.authorize
+            self.create_success_response = ReactApi.create_success_response
+            self.create_response = ReactApi.create_response
+            self.get_site_name = lambda: "some-site-name"
+            self.APP_PACKAGE_NAME = "foursight"
+
+    class MockChaliceRequest:
```

Comment:
This seems a better use of an `__init__` method because presumably instances will get different requests. But you could still probably put this class at file top level.